MAINTAINERS.yml

```diff
@@ -3180,9 +3180,13 @@ Intel Platforms (X86):
     - tbursztyka
     - laurenmurphyx64
   files:
-    - boards/x86/
+    - boards/intel/intel_adl/
+    - boards/intel/intel_ehl/
+    - boards/intel/intel_rpl/
     - dts/x86/intel/
-    - soc/x86/
+    - soc/intel/atom/
```

Comment:
similar, probably `soc/intel`, unless of course @jhedberg and others don't want to me pinged for any changes at this level.

Reply:
see my comment above